client/src/Components/Navigation/Header.js

```diff
@@ -1,28 +1,28 @@
-import React, { useEffect, useState } from "react";
-import { useDispatch } from "react-redux";
-import { useHistory, Link, useLocation } from "react-router-dom";
-import decode from "jwt-decode";
+import React, { useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { useHistory, Link, useLocation } from 'react-router-dom'
+import decode from 'jwt-decode';
 
 const Header = () => {
-  const dispatch = useDispatch();
-  const history = useHistory();
-  const location = useLocation();
-  const [user, setUser] = useState(JSON.parse(localStorage.getItem("profile")));
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const location = useLocation()
+  const [user, setUser] = useState(JSON.parse(localStorage.getItem('profile')))
```

Comment:
I remember someone created a constant for local storage keys. Please use that here